src/PublishPort.jsx

```diff
@@ -57,8 +102,12 @@
                             step={1}
                             min={1}
                             max={65535}
+                            validated={validationFailed?.containerPort ? "error" : "default"}
+                            onFocus={() => utils.validationClear(validationFailed, "containerPort", onValidationChange)}
+                            onBlur={() => onValidationChange(validatePublishPort(item, "containerPort", validationFailed))}
```

Comment:
These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1354-20230907-093411-a9f37bd4-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)